### PR TITLE
Fix syntax for 'infinity' in Go / math functions / `Tan()` - Closes #3088

### LINKED ISSUE #3088: Correcting the way infinity is represented in Go math functions documentation
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

---
Title: 'Documentation Update for Tan Function in Go' 
Description: 'Correcting the way infinity is represented in Go math functions documentation.'
Subjects: 
  - 'Code Foundations'
  - 'Computer Science'
Tags: 
  - 'Numbers'
  - 'Hyperbolic Functions'
  - 'Mathematics'
  - 'Documentation'
CatalogContent: 
  - 'paths/computer-science'
---

In the documentation found under `docs/content/go/concepts/math-functions/terms/tan/tan.md`, there's a discrepancy in the way the results for certain inputs to the `Tan` function are represented.This generates a erro in go `undefined: Inf'.

**Current Representation**:
- The result of `Tan(-Inf)` is `NaN`
- The result of `Tan(+Inf)` is `NaN`
- The result of `Tan(NaN)` is `NaN`

**Proposed Update**:
- The result of `Tan(math.Inf(+1))` is `NaN`
- The result of `Tan(math.Inf(-1))` is `NaN`
- The result of `Tan(NaN)` is `NaN`

This change aligns the representation with the correct way to specify positive and negative infinity in Go.

## Codebyte Example (if applicable)

We can currently support:

- Python
- JavaScript
- Ruby
- C++
- C#
- Go
- PHP

See [content-standards.md](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) for more details!

```go
// Example showing the correct representation
package main

import (
	"fmt"
	"math"
)

func main() {
	fmt.Println(math.Tan(math.Inf(+1))) // Outputs: NaN
	fmt.Println(math.Tan(math.Inf(-1))) // Outputs: NaN
	fmt.Println(math.Tan(math.NaN()))   // Outputs: NaN
}


### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [X] Issue is assigned

### PR DESCRIPTION
**Pull request by douglas-narcizo**
### Description

This PR is just a fix to the way that *'infinity'* function is used in the examples for the **Tan()** entry - as appointed on the opened *issue*.

*(Motivation: to accomplish a 'first issue' full solving cycle, as part of my learning on Codecademy's **"Learn Git & GitHub"** course.)*

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/go/concepts/math-functions/terms/tan/tan.md [hunk 1]
```diff
@@ -65,13 +65,8 @@ import (
...
-  negativeInfinity := math.Inf(-1)
-
-  tanPositiveInf := math.Tan(positiveInfinity)
-  tanNegativeInf := math.Tan(negativeInfinity)
-
-  fmt.Printf("The tangent of positive infinity: %f\n", tanPositiveInf)
-  fmt.Printf("The tangent of negative infinity: %f\n", tanNegativeInf)
+	fmt.Println(math.Tan(math.Inf(+1))) // Outputs: NaN
+	fmt.Println(math.Tan(math.Inf(-1))) // Outputs: NaN
+	fmt.Println(math.Tan(math.NaN()))   // Outputs: NaN
```
Comment (SSwiniarski) [inline content/go/concepts/math-functions/terms/tan/tan.md:70]:
```suggestion
	fmt.Println(math.Tan(math.Inf(+1)))
	fmt.Println(math.Tan(math.Inf(-1)))
	fmt.Println(math.Tan(math.NaN()))
```
We don't need the output comment on a codebyte

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@douglas-narcizo I did a first review and had one comment.

File: content/go/concepts/math-functions/terms/tan/tan.md (hunk 1 above)
Comment (douglas-narcizo) [reply content/go/concepts/math-functions/terms/tan/tan.md]: Very true!! I had just fixed it and removed the unnecessary comments, and committed the adjusted branch.
Thanks for the advice!

**Review by SSwiniarski — APPROVED** [review]
@douglas-narcizo I made a minor edit and it is ready for a second review.

File: content/go/concepts/math-functions/terms/tan/tan.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ result := math.Tan(angle)
 
 Where `result` is the tangent value of `angle`, returned as a float, except under the following circumstances:
 
-- The result of `Tan(-Inf)` is `NaN`
-- The result of `Tan(+Inf)` is `NaN`
+- The result of `Tan(math.Inf(+1))` is `NaN`
+- The result of `Tan(math.Inf(-1))` is `NaN`
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/tan/tan.md:27]: I don't this should be changed. I don't really see a problem with the shorthand in the syntax section, I believe almost all of our entries use this format. Additionally, if we were to follow through with this approach then we would need to change the last line to Tan(math.NanN()). I would suggest reverting these changes

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@douglas-narcizo Just added one comment.

File: content/go/concepts/math-functions/terms/tan/tan.md (hunk 2 above)
Comment (douglas-narcizo) [reply content/go/concepts/math-functions/terms/tan/tan.md:27]: Sure, no problem! I apologize for this… just found the open _issue_ listed when I was looking for an available _'first issue'_ to solve, as part of Codecademy's **"Learn Git & GitHub"** course.
Also, thinking in a positive way on this mistake, that's a good opportunity to ask what comes next… should I terminate this PR? Or the team will do this, and close the corresponding _issue_?

**Comment by douglas-narcizo** [comment]
@caupolicandiaz I replied to your comment _(not sure if my reply was made in the right way or place)_… but I believe that this PR and its corresponding issue should be closed.
My question was about the roles in such case… should I close the PR and the team close the issue?
I wait for your instructions. I apologize for any inconvenience and thanks for understanding.

**Comment by caupolicandiaz** [comment]
> @caupolicandiaz I replied to your comment _(not sure if my reply was made in the right way or place)_… but I believe that this PR and its corresponding issue should be closed.
> My question was about the roles in such case… should I close the PR and the team close the issue?
> I wait for your instructions. I apologize for any inconvenience and thanks for understanding.

Apologies for the confusion. This is all set. I'll close it out. Thanks

**Pull request closed without merging**